darts/timeseries.py

```diff
@@ -445,33 +467,29 @@ def from_values(values: np.ndarray,
             A Numpy array of values for the TimeSeries. Both 2-dimensional arrays, for deterministic series,
             and 3-dimensional arrays, for probabilistic series, are accepted. In the former case the dimensions
             should be (time, component), and in the latter case (time, component, sample).
-        fill_missing_dates
-            Optionally, a boolean value indicating whether to fill missing dates with NaN values. This requires
-            either a provided `freq` or the possibility to infer the frequency from the provided timestamps.
-            Inferring the frequency and resampling the data can induce a significant performance overhead.
-        freq
-            Optionally, a string representing the frequency of the Pandas DataFrame. This is useful in order to fill
-            in missing values if some dates are missing and `fill_missing_dates` is set to `True`.
         columns
             Columns to be used by the underlying pandas DataFrame.
+        sort
+            Whether to sort over the time axis when creating the TimeSeries. Can be set to False for improving
+            performance in cases where the provided time axis is already sorted.
 
         Returns
         -------
         TimeSeries
             A TimeSeries constructed from the inputs.
         """
-
         time_index = pd.RangeIndex(0, len(values), 1)
-
         values_ = np.reshape(values, (len(values), 1)) if len(values.shape) == 1 else values
+
         return TimeSeries.from_times_and_values(times=time_index,
                                                 values=values_,
-                                                fill_missing_dates=fill_missing_dates,
-                                                freq=freq,
-                                                columns=columns)
+                                                fill_missing_dates=False,
```

Comment:
What is the role of fill_missing_date in from_values function if it is set to false here?

Reply:
I removed it from `from_values()` because it doesn't make sense to have it here (we are creating the time axis here anyway, so there's no "missing value").